packages/-ember-data/tests/integration/adapter/find-test.js

```diff
@@ -178,17 +214,26 @@ module('integration/adapter/find - Finding Records', function(hooks) {
       })
     );
 
-    let store = this.owner.lookup('service:store');
+    const store = this.owner.lookup('service:store');
+    const promises = [store.findRecord('person', '1'), store.findRecord('person', '2')];
 
-    assert.expectAssertion(() => {
-      run(() => {
-        store.findRecord('person', '1');
-        store.findRecord('person', '2');
-      });
-    }, /You made a 'findMany' request for 'person' records with ids '\[1,2\]', but the adapter's response did not have any data/);
+    try {
+      await all(promises);
+    } catch (error) {
+      const expectedMessageRegex = /You made a 'findMany' request for 'person' records with ids '\[1,2\]', but the adapter's response did not have any data/;
+
+      assert.ok(expectedMessageRegex.test(error.message));
```

Comment:
same as above, label for the assertion and likely shift to strictEquals string comparison